components/automate-ui/src/app/services/node-details/node-runs.service.ts

```diff
@@ -59,6 +59,21 @@ export class NodeRunsService {
       });
   }
 
+  getNodeRunsByID(nodeId: string) {
```

Comment:
Please use strong types on this function and all the others in this file:
```suggestion
  getNodeRunsByID(nodeId: string): Promise<AbridgedNodeRun[]> {
```
Besides the obvious benefits right here, it also gives types in the consumer NodeNoRunIdResolverService so one can understand the data there much better.